testng-core/src/main/java/org/testng/thread/Async.java

```diff
@@ -0,0 +1,46 @@
+package org.testng.thread;
```

Comment:
As long as we release a all in one jar, use internal package for everything that is not part of the api/spi.
Here, org.testng.internal.thread looks better.

Reply:
This was a slip. My bad. It was meant to be in the `internal` package. Thanks for pointing it out. Fixed it.